errctx/errctx.go

```diff
@@ -0,0 +1,174 @@
+package errctx
```

Comment:
I see lots of consonants in this package name, not sure if I can come up with something better. Have you considered calling it simply `errors`? Or `github.com/remind101/pkg/x/errors` maybe..

Reply:
I was trying to think of a better package name as well. Perhaps something that signifies it should be used in an http handler? 
```
httperrors
httpxerrors
```

Or if I was to name it `errors` I think I would want to expose all the same functionality as `pkg/errors` so people wouldn't get too confused about which package to use.

Reply:
what if you move it into `httpx`? `github.com/remind101/pkg/httpx/errors`?